packages/form-field-scripts/src/logBoxen.js

```diff
@@ -0,0 +1,35 @@
+const { default: boxen } = await import('boxen')
+
+const logBoxenError = ({ title, description }) => {
```

Comment:
why is this error needed

Reply:
Its used to log errors that is emitted by webpack, if not needed I shall remove it.

Reply:
not needed da